[2023-01-29T21:16:22.523+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: data_pipeline_dag.extract_data_task_data_scientist manual__2023-01-29T21:15:20.067421+00:00 [queued]>
[2023-01-29T21:16:22.578+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: data_pipeline_dag.extract_data_task_data_scientist manual__2023-01-29T21:15:20.067421+00:00 [queued]>
[2023-01-29T21:16:22.579+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-01-29T21:16:22.579+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 3
[2023-01-29T21:16:22.579+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-01-29T21:16:22.759+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): extract_data_task_data_scientist> on 2023-01-29 21:15:20.067421+00:00
[2023-01-29T21:16:22.824+0000] {standard_task_runner.py:55} INFO - Started process 2282 to run task
[2023-01-29T21:16:22.835+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline_dag', 'extract_data_task_data_scientist', 'manual__2023-01-29T21:15:20.067421+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpjddt8fii']
[2023-01-29T21:16:22.839+0000] {standard_task_runner.py:83} INFO - Job 22: Subtask extract_data_task_data_scientist
[2023-01-29T21:16:22.953+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:249 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-01-29T21:16:23.218+0000] {task_command.py:388} INFO - Running <TaskInstance: data_pipeline_dag.extract_data_task_data_scientist manual__2023-01-29T21:15:20.067421+00:00 [running]> on host 84fb6dd1372f
[2023-01-29T21:16:23.483+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=abdessamad
AIRFLOW_CTX_DAG_ID=data_pipeline_dag
AIRFLOW_CTX_TASK_ID=extract_data_task_data_scientist
AIRFLOW_CTX_EXECUTION_DATE=2023-01-29T21:15:20.067421+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-29T21:15:20.067421+00:00
[2023-01-29T21:17:21.216+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-01-29T21:17:21.233+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=data_pipeline_dag, task_id=extract_data_task_data_scientist, execution_date=20230129T211520, start_date=20230129T211622, end_date=20230129T211721
[2023-01-29T21:17:21.297+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-01-29T21:17:21.358+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
